libs/langchain/langchain/chat_loaders/imessage.py

```diff
@@ -107,8 +107,13 @@ def lazy_load(self) -> Iterator[chat_loaders.ChatSession]:
             ) from e
```

Comment:
Does it make sense to keep the error above as an OperationalError, but expand the message?